modules/broker-auth/src/main/java/io/ballerina/messaging/broker/auth/authorization/handler/AuthorizationHandler.java

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
Do we need a separate package for handler class? Shall we move it to 
`io.ballerina.messaging.broker.auth.authorization` package?

Reply:
Fixed in 810c187